Makefile

```diff
@@ -131,15 +131,15 @@ install:
 
 release-install:
 	@$(call print, "Installing release lnd and lncli.")
-	env CGO_ENABLED=0 $(GOINSTALL) -v -trimpath -ldflags="$(RELEASE_LDFLAGS)" -tags="$(RELEASE_TAGS)" $(PKG)/cmd/lnd
-	env CGO_ENABLED=0 $(GOINSTALL) -v -trimpath -ldflags="$(RELEASE_LDFLAGS)" -tags="$(RELEASE_TAGS)" $(PKG)/cmd/lncli
+	env CGO_ENABLED=0 $(GOINSTALL) -v -trimpath $(RELEASE_LDFLAGS) -tags="$(RELEASE_TAGS)" $(PKG)/cmd/lnd
+	env CGO_ENABLED=0 $(GOINSTALL) -v -trimpath $(RELEASE_LDFLAGS) -tags="$(RELEASE_TAGS)" $(PKG)/cmd/lncli
 
 # Make sure the generated mobile RPC stubs don't influence our vendor package
 # by removing them first in the clean-mobile target.
 release: clean-mobile
 	@$(call print, "Releasing lnd and lncli binaries.")
 	$(VERSION_CHECK)
-	./scripts/release.sh build-release "$(VERSION_TAG)" "$(BUILD_SYSTEM)" "$(RELEASE_TAGS)" "$(RELEASE_LDFLAGS)"
+	./scripts/release.sh build-release "$(VERSION_TAG)" "$(BUILD_SYSTEM)" "$(RELEASE_TAGS)" $(RELEASE_LDFLAGS)
```

Comment:
`build-release` also has `-ldflags` supplied.

Reply:
I was checking the output of what was being called and did not noticed that `release.sh` already adds it. I will add it only where it was missing (`install` target)